pkg/eniconfig/eniconfig.go

```diff
@@ -35,8 +35,19 @@ import (
 )
 
 const (
-	eniConfigAnnotationDef = "k8s.amazonaws.com/eniConfig"
-	eniConfigDefault       = "default"
+	defaultEniConfigAnnotationDef = "k8s.amazonaws.com/eniConfig"
```

Comment:
Can we store the value being used in the `ENIConfigController`?  The `New*` methods can read the ENVs or use the defaults.  Then the ENV lookup doesn't happen on every request.  I don't think they should change during the execution of the program so that shouldn't be an issue.

Reply:
made a change to get the label and annotation definition only once.